packages/react/src/components/Popup/Popup.tsx

```diff
@@ -478,17 +470,15 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
         rtl={rtl}
         unstable_pinned={unstable_pinned}
         targetRef={this.rightClickReferenceObject || target || this.triggerRef}
-        children={this.renderPopperChildren.bind(this, popupPositionClasses, rtl, accessibility)}
+        children={this.renderPopperChildren(rtl, accessibility)}
```

Comment:
why were we previosly binding this?

Reply:
I don't have any idea as it's obviously more cleaner now. The result of `.bind()` will be the same as it creates a new function...